.github/workflows/ci.yml

```diff
@@ -18,3 +18,46 @@ jobs:
           path: fuzzing/corpus
           key: fuzzing
       - run: npm run fuzz
+
+  verify:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: Install Python
+        uses: actions/setup-python@v2
+        with: { python-version: 3.9 }
+
+      - name: Install Java
+        uses: actions/setup-java@v1
+        with: { java-version: "11", java-package: jre }
+
+      - name: Install Certora CLI
+        run: pip3 install certora-cli==7.6.3
+
+      - name: Install Solidity
+        run: |
+          wget https://github.com/ethereum/solidity/releases/download/v0.8.19/solc-static-linux
```

Comment:
We need a newer version of the solidity compiler here, because our solidity files specify the 0.8.23 compiler.